### ISSUE TITLE
[Bug]: service "php" can't be used with `extends` as it declare `depends_on`

### ISSUE THREAD

**Issue by fashxp** [pimcore/skeleton#184]
### Expected behavior

docker compose works with shipped docker-compose.yaml

### Actual behavior

fails with error service "php" can't be used with `extends` as it declare `depends_on`

see also https://github.com/docker/compose/issues/11544

### Steps to reproduce

create project and run `docker compose up -d` with latest docker compose version

**Comment by gander** [comment]
```
service "php" can't be used with `extends` as it declare `depends_on`
```

1. Reason: service `test-php` extends `php` that declare `depends_on`
2. Error occurs since **Docker Compose** `v2.24.6`
3. Docker Compose documentation: [Restrictions](https://docs.docker.com/compose/compose-file/05-services/#restrictions)
4. **CLOSED**, they won't "fix" it: [docker/compose#11544](https://github.com/docker/compose/issues/11544)

**Comment by fashxp** [comment]
but there is a PR https://github.com/compose-spec/compose-spec/pull/470 that might fix this again...

**Comment by jdreesen** [comment]
Which was merged now!

**Comment by jdreesen** [comment]
[compose/issues#11544](https://github.com/docker/compose/issues/11544#issuecomment-2017666764) is now resolved and it should work again with `v2.26.0`.
